src/ImporterController.php

```diff
@@ -22,6 +22,11 @@ class ImporterController extends Controller
      */
     public $importModel;
 
+    /**
+     * @var int
+     */
+    public $user_id;
```

Comment:
does this get set anywhere?

Reply:
It will be set at beforeAction() method of ImportNetworkTariffController (samurai project).

Reply:
@NgocTuanTruong  could this use pull the user directly?
This code would have access to `Yii::$app` right? Why not make this package responsible for this?